src/service.rs

```diff
@@ -1128,6 +1175,10 @@ impl CollectionInvitationManager {
         pubkey: &[u8],
         access_level: CollectionAccessLevel,
     ) -> Result<()> {
+        // TODO: change argument type to &[u8; 32]
```

Comment:
Plan on fixing it before it gets in, or is it just a note for later?

Reply:
It's a public API, so this is a note for the next major release. I can clarify the comment if you want.

Reply:
I can do that - any preferences for naming?

Reply:
Not really. :)